### PR TITLE
Doc updates from PR

### PR DESCRIPTION
**Pull request by rjmurillo**
Related to PR #4

### CODE REVIEW

File: docs/controls/WebView.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ When compared to the UWP **WebView** control, the **WebView** controls for WPF a
```
Comment (rjmurillo) [inline docs/controls/WebView.md]: @mcleanbyron @normesta  In prerequisites we define the minimum OS and SDK version; however, @nmetulev pointed out that there is a standard form of this in the doc template. I've included it on L312 for your review

File: docs/controls/WebView.md [hunk 2]
```diff
@@ -309,6 +309,16 @@ Scripts in the web view content can use **window.external.notify** with a string
 
 You can use the [Settings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.settings) property (of type [WebViewControlSettings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrolsettings) to control whether JavaScript and IndexedDB are enabled. For example, if you use a web view to display strictly static content, you might want to disable JavaScript for best performance.
 
- 
+ ## Requirements
```
Comment (rjmurillo) [inline docs/controls/WebView.md]: @nmetulev Where should we put the information regarding the high DPI issues / workarounds (setting app.config values for WinForms, setting manifest information for WinForms and WPF)?

Comment (nmetulev) [reply docs/controls/WebView.md]: Using a NOTE or IMPORTANT seems appropriate for this. Here is the syntax:

```
> [!NOTE]
Some note

> [!IMPORTANT]
Some important note

> [!WARNING]
Some warning note
```

For example, here how it renders in docs:
image

Comment (rjmurillo) [reply docs/controls/WebView.md]: @normesta I'll write something up and include it here for review

Comment (rjmurillo) [reply docs/controls/WebView.md]: @nmetulev @normesta Included in 30ad73c

**Pull request merged**